docs/connecting-data/connecting-to-apis-and-external-data/servicenow-api.md

```diff
@@ -1,7 +1,150 @@
----
-description: How to Connect DataDistillr to the ServiceNow API
----
-
 # ServiceNow API
 
-Details coming soon!
\ No newline at end of file
+## Creating a ServiceNow account
+Set up an account with [ServiceNow](https://www.servicenow.com/).
+
+
+### Rate Limits
+To prevent excessive inbound REST API requests, set rules that limit the number of inbound REST API requests processed per hour. 
```

Comment:
Are there specific rate limits for ServiceNow? This seems like a general comment that could apply to any API.

Reply:
Rate limits are set by the admin of the account

Reply:
At the very least provide the following link: https://docs.servicenow.com/bundle/sandiego-application-development/page/integrate/inbound-rest/concept/inbound-REST-API-rate-limiting.html